templates/data-sources/resource_policies.md.tmpl

```diff
@@ -0,0 +1,13 @@
+# {{.Type}}: {{.Name}}
+
+`{{.Name}}` returns all resource policies in an organization.
+
+-> **NOTE**: Resource Policies are currently in Public Preview. You will need to
```

Comment:
same comment as on first file

Reply:
Changed in a8851acdb8c44933b67c874a3160f5b6938a5ba2